Gems/ScriptCanvas/Code/Editor/View/Windows/Tools/UpgradeTool/VersionExplorer.cpp

```diff
@@ -301,75 +302,47 @@ namespace ScriptCanvasEditor
             return "";
         }
 
+        auto fileIoBase = AZ::IO::FileIOBase::GetInstance();
+        auto settingsRegistry = AZ::SettingsRegistry::Get();
+
         QDateTime theTime = QDateTime::currentDateTime();
         QString subFolder = theTime.toString("yyyy-MM-dd [HH.mm.ss]");
 
-        AZStd::string backupPath = AZStd::string::format("@devroot@/ScriptCanvas_BACKUP/%s", subFolder.toUtf8().data());
-        char backupPathCStr[AZ_MAX_PATH_LEN] = { 0 };
-        AZ::IO::FileIOBase::GetInstance()->ResolvePath(backupPath.c_str(), backupPathCStr, AZ_MAX_PATH_LEN);
-        backupPath = backupPathCStr;
+        AZ::IO::FixedMaxPath projectUserPath;
+        settingsRegistry->Get(projectUserPath.Native(), AZ::SettingsRegistryMergeUtils::FilePathKey_ProjectUserPath);
+
+        AZ::IO::FixedMaxPath backupPath = projectUserPath / "ScriptCanvas_BACKUP" / subFolder.toUtf8().data();
```

Comment:
Now that I see this, I would check other places where `ScriptCanvas_BACKUP` path was being used and make sure they all use this `ProjectUserPath`.

Reply:
There were a couple of other places in [UpgradeTool.cpp](https://github.com/o3de/o3de/blob/6be709aa5369d77f7101967bdf48cfc3e2b807ad/Gems/ScriptCanvas/Code/Editor/View/Windows/MainWindow.cpp#L747) and [MainWindow.cpp](https://github.com/o3de/o3de/blob/6be709aa5369d77f7101967bdf48cfc3e2b807ad/Gems/ScriptCanvas/Code/Editor/View/Windows/MainWindow.cpp#L747).

I will update those to use the user directory